server/channels/testlib/assertions.go

```diff
@@ -7,33 +7,31 @@ import (
 	"encoding/json"
 	"io"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 // AssertLog asserts that a JSON-encoded buffer of logs contains one with the given level and message.
 func AssertLog(t *testing.T, logs io.Reader, level, message string) {
```

Comment:
Should we add a `t.Helper` to these methods?

Reply:
Yeah good point